templates/epic.xml.jinja2

```diff
@@ -104,7 +104,7 @@
   <documentation level="10">
     ## Central tracking detectors
   </documentation>
-  <include ref="compact/tracking_config.xml"/>
+  <include ref="compact/tracker_ecce.xml"/>
```

Comment:
we've got an `epic` tracker now ;-)

Reply:
so do we want `tracker_epic.xml `or `epic_tracker.xml`?

Reply:
Existing convention seems to say `tracker.xml`

Reply:
I prefer my trackers big-endian.

Reply:
I wanted more granularity here to make it easier to switch/test different high-level configurations (as in with very different parametrization). The reason it was called `*_ecce` is because it is the configuration from the proposal. Maybe calling it `tracker_reference.xml` makes the most sense.